apps/files/tests/test_utils_.py

```diff
@@ -251,8 +251,8 @@ def test_instanciate_without_data(self):
         with NamedTemporaryFile() as file_:
             file_.write(json.dumps(data))
             file_.flush()
-            pje = PackageJSONExtractor(file_.name)
-            assert pje.data == data
+            mje = ManifestJSONExtractor(file_.name)
+            assert mje.data == data
```

Comment:
Should these changes go in #889? Perhaps it makes sense to merge the two PRs but leave it as two separate commits?